src/processors/default/processPlatformTracks/processPlatformTracks.ts

```diff
@@ -59,14 +60,21 @@ const processorFunction = (platform: MusicPlatform) => async (nfts: NFT[], clien
 
   for (const nftFactory of nftFactories) {
     const factoryNFTs = nftsByFactoryId[nftFactory.id];
+    const nftToTrackIdSource: NFTtoTrackIdSource = {
+      nfts: factoryNFTs,
+      dbClient: clients.db,
+      apiTracksByNFT: apiTracksByNFT,
+      contract: nftFactory
+    }
+
     let nftFactoryType: MusicPlatformTypeConfig;
     try {
       nftFactoryType = getNFTFactoryType(nftFactory, platformType);
       const {
         newTrackIds,
         trackMapping,
         existingTrackIds,
-      } = await getTrackInputs(nftFactoryType.mappers.mapNFTsToTrackIds, factoryNFTs, clients.db, apiTracksByNFT);
+      } = await getTrackInputs(nftFactoryType.mappers.mapNFTsToTrackIds, nftToTrackIdSource);
```

Comment:
put dbClient back into getTrackInputs parameters, since it is still needed in getTrackInputs even though not within the mapper

```suggestion
      } = await getTrackInputs(nftFactoryType.mappers.mapNFTsToTrackIds, nftToTrackIdSource, clients.db);
```